### DISCUSSION TITLE
thanhle.blog/en/blog/giao-tiep-hieu-qua-cho-dev

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#44] [Ideas]
# thanhle.blog/en/blog/giao-tiep-hieu-qua-cho-dev

Cách mình cải thiện giao tiếp khi làm dev

https://thanhle.blog/en/blog/giao-tiep-hieu-qua-cho-dev

**Comment by haulinh** [comment]
Bài viết hay lắm anh. Ai hướng nội đọc bài này hợp lý nè. Để em share với bạn bè

**Comment by 2dtrung** [comment]
Bài viết gãi đúng chỗ em đang ngứa. Thật sự em là người luôn tìm các giải pháp cho bản thân mỗi khi em cần giải quyết một vấn đề mà mình đang không tốt. Và bài viết này đã cover gần như đầy đủ vấn đề về giao tiếp để em có thể áp dụng trong team mình

**Reply by thanhlmm** [reply]
Yeah, lúc đầu hơi khó xíu nhưng chỉ cần tập 3-4 lần là sẽ giao tiếp hiệu quả ngay. Ít nhất là trong lúc chat hoặc viết email :))))

Còn nói thì phải có bạn gái mới giỏi dc :)))
